rls/src/main/java/io/grpc/rls/CachingRlsLbClient.java

```diff
@@ -530,6 +544,7 @@ final class DataCacheEntry extends CacheEntry {
           refCountedChildPolicyWrapperFactory
               .createOrGet(response.targets());
       long now = ticker.read();
+      minEvictionTime = now + TimeUnit.SECONDS.toNanos(5);
```

Comment:
Make "5" a constant.

Reply:
Made the whole `TimeUnit.SECONDS.toNanos(5)` a constant